### DISCUSSION TITLE
Openblas + windows

### DISCUSSION THREAD

**Discussion by mgutierrezp** [OpenMathLib/OpenBLAS#4964] [Q&A]
Hi all,

I'm trying to install the libopenblas.dll in a windows box with no admin rights; so I cannot overwrite the Rblas.dll. I tried different methods by modifying the PATH env var and also by the dyn.load command but no luck.

Is it possible what I am try to achieve or admin rights are mandatory needed?

Regards

**Comment by SanPen** [comment]
To "install" you just need to modify the PATH. I belive there's a USER PATH variable where you can include the folder where you store `libopenblas.dll`

**Comment by mgutierrezp** [comment]
thanks for your reply. I tried that and in different combinations. No luck

**Comment by martin-frbg** [comment]
I think it used to be easier to manipulate dll load paths in earlier versions of Windows (which also had security implications of course). Guess the best solution is to perform a separate installation of R in your user folder and replace the dll there, as detailed e.g. by  https://gist.github.com/seananderson/08a51e296a854f227a908ddd365fb9c1

**Comment by mgutierrezp** [comment]
Thanks Martin. I already did read that post. Unfortunately corporative
rules don't allow this. The only way I see is by having admin rights. I
will assume that.

Thanks everyone for the help and tips..

M
